app/uk/gov/hmrc/agentoverseasfrontend/models/SessionDetails.scala

```diff
@@ -36,6 +36,19 @@ object SessionDetails {
     SessionDetails(id, authProviderId)
   }
 
-  implicit val localDateTimeFormats: Format[LocalDateTime] = MongoJavatimeFormats.localDateTimeFormat
+  final val localDateTimeReads: Reads[LocalDateTime] =
```

Comment:
Need to verify if we're changing the format here, we don't want to be storing dates as Long

Reply:
Looks good locally